### ISSUE TITLE
Typo in Parameter Placeholder

### ISSUE THREAD

**Issue by timstringer** [Corcules/KMlink#9]
The placeholder text for the Parameter field contains a typo ("**Optionnal** parameter"). Thanks!

**Comment by Corcules** [comment]
Good point Tim, thank you. It's coming from a quick manual translation. In french we have two n.
